exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
```

Comment:
given that problem-specifications is currently at 1.0.1, why not say adapted from 1.0.1?